src/cat-data-service/src/service/api/mod.rs

```diff
@@ -0,0 +1,94 @@
+//! Catalyst Data Service API Definition
+//!
+//! This defines all endpoints for the Catalyst Data Service API.
+//! It however does NOT contain any processing for them, that is defined elsewhere.
+use health::HealthApi;
+use poem_openapi::{
+    param::Query, payload::PlainText, ContactObject, LicenseObject, OpenApi, OpenApiService,
+    ServerObject,
+};
+
+use crate::settings::API_URL_PREFIX;
+
+mod health;
+mod test_endpoints;
+
+/// The name of the API
+const API_TITLE: &str = "Catalyst Data Service";
+
+/// The version of the API
+const API_VERSION: &str = "1.2.0";
+
+/// Get the contact details for inquiring about the API
+fn get_api_contact() -> ContactObject {
+    ContactObject::new()
+        .name("Project Catalyst Team")
```

Comment:
Not possible as far as I can tell because `ContactObject::new` needs to be used to create a Contact Object, which you can't do on a const.

The individual values of the fields are not exported or re-used anywhere, so I don't see a lot of value in defining:

`const API_CONTACT_NAME: &str = "Project Catalyst Team";`

and then

```
ContactObject::new().name(API_CONTACT_NAME)...
```
It seems needlessly verbose.